extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -15,4 +15,11 @@
  */
 
  amp-sticky-ad {
+ 	position: fixed;
+ 	bottom: 0;
+ 	left: 0 !important;
+ 	z-index: 1;
+ 	width: 100%;
+ 	display: none;
+ 	max-height: 100px;
```

Comment:
From what I understand, the `position: fixed`, `max-height:100px` `bottom:0` should not be changed. I am not sure about `z-index:1` but it definitely can't have a value less than 1.